models/vision/detection/yolo/yolov3_postproc.cc

```diff
@@ -15,14 +15,30 @@
 // limitations under the License.
 // =============================================================================
 
+#include <dlfcn.h>
```

Comment:
maybe we should make a separate version? This one is for CPU postprocessing.
You can make a yolov3_postproc_ipu.cc

Reply:
and here you shouldn't include IPU header files.
you can use ODLA APIs

Reply:
You can implement odla_NMS inside    [ODLA/platforms/odla_popart/odla_nn.cc](https://github.com/alibaba/heterogeneity-aware-lowering-and-optimization/blob/master/ODLA/platforms/odla_popart/odla_ops_nn.cc) 

Refer to [erf example](https://github.com/alibaba/heterogeneity-aware-lowering-and-optimization/blob/9a6a6831cc893b655e3ed30f240e813c07cba0f6/ODLA/platforms/odla_popart/odla_ops.cc#L102)